run_ansible.sh

```diff
@@ -129,6 +129,7 @@ pipenv_init() {
           pip3 install pipenv --user || exit 1
           echo '------ enable pip --user installations to be accesible'
           echo -en '\\n[[ -d $HOME/.local/bin ]] && {\\n  PATH=\"$HOME/.local/bin:$PATH\"\\n}' >> $HOME/.bashrc
```

Comment:
We should check for the existence, and quote our vars. 

```suggestion
          if [[ -e "$HOME/.bashrc" ]]; then
              echo -en '\\n[[ -d $HOME/.local/bin ]] && {\\n  PATH=\"$HOME/.local/bin:$PATH\"\\n}' >> $HOME/.bashrc
              source "$HOME/.bashrc"
          else
              echo ".bashrc not found.  Pipenv (and other user installed pip apps) may not work."
          fi
```